pkg/v13patch1/chartconfig/desired.go

```diff
@@ -0,0 +1,128 @@
+package chartconfig
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/errors/guest"
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/label"
+	"github.com/giantswarm/cluster-operator/pkg/v13patch1/key"
+)
+
+const (
+	chartConfigAPIVersion           = "core.giantswarm.io"
+	chartConfigKind                 = "ChartConfig"
+	chartConfigVersionBundleVersion = "0.4.0"
```

Comment:
@giantswarm/sig-operator this is still odd. At some point this should be part of a release and not magically hidden somewhere deep down in the code. Just saying.

Reply:
Yes 100%. Once we have managed apps for customers the next step is migrating from chartconfig to app CRs. The apps will move to the release CR and we won't have all this config hidden away in cluster-operator.